battetl/load/batt_db_test_helper.py

```diff
@@ -97,45 +103,6 @@ def read_last_row(self, target_table: str, pk_col_name: str) -> tuple:
 
         return last_row
 
-    def perform_insert(self, target_table: str, dict_to_load: dict, pk_id_col: str) -> int:
-        """
-        Inserts the values from the dict_to_load into the `target_table`. Assumes
-        columns are keys. Drops any empty keys.
-
-        Parameters
-        ----------
-        target_table : str
-            The table to perform the insert into.
-        dict_to_load : dict
-            The dictionary containing the key value pairs to load into the target table.
-        pk_id_col : str
-            The name of the column for the primary key id.
-
-        Returns
-        -------
-        pk_id : int
-            The primary key id for the newly inserted row. Returns None if issue with inserting rows.
-        """
-
-        # Remove any empty entries from upload dict
-        {k: v for k, v in dict_to_load.items() if v}
-
-        insert_statement = psycopg2.sql.SQL(
-            "INSERT INTO {table} ({cols}) VALUES ({vals}) RETURNING {pk_id}").format(
-            table=psycopg2.sql.Identifier(target_table),
-            cols=psycopg2.sql.SQL(', ').join(
-                map(psycopg2.sql.Identifier, list(dict_to_load.keys()))),
-            vals=psycopg2.sql.SQL(', ').join(
-                psycopg2.sql.Placeholder() * len(dict_to_load)),
-            pk_id=psycopg2.sql.Identifier(pk_id_col)
-        )
-        with self.conn.cursor() as cursor:
-            cursor.execute(insert_statement, list(dict_to_load.values()))
-            result = cursor.fetchone()
-        if result:
-            pk_id = result[0]
-
-        return pk_id
 
     def load_df_to_db(self, df: pd.DataFrame, target_table: str) -> int:
```

Comment:
Yeah, it's just a wrapper around the private parent method. We could call the private parent method directly in the code, but then it would not have access to the class instance variables which are needed for the upload.